temp_model.py

```diff
@@ -1 +1,124 @@
+import pandas as pd 
+import numpy as np
+import matplotlib.pyplot as plt
+from numpy.fft import fft, ifft, fftshift, fftfreq
+from scipy.signal import max_len_seq
 
+import rospy
+from std_msgs.msg import Bool as Bool_msg
+from std_msgs.msg import Empty as Empty_msg
+from std_msgs.msg import Float64 as Float64_msg
+from std_msgs.msg import String as String_msg
```

Comment:
```suggestion
import std_msgs.msg as msg
```
and changing `Bool_msg` to `msg.Bool` etc. achieves the same thing in 1 line.

Reply:
cool . I should learn this way to comment. Thx